react-interactive/src/Menue/Page.tsx

```diff
@@ -45,29 +45,10 @@ export interface IProps {
 const Page: React.FunctionComponent<IProps> = (props) => {
     const [hover, setHover] = React.useState<boolean>(false);
     const context = React.useContext(Context)
-    const location = useLocation();
-
-    const isPathActive = () => {
-        // Use Name as the base check and see if the current path starts with this base.
-        return location.pathname.startsWith(`${context.homePath}${props.Name}`)
-            || isOtherPathActive();
-    }
-
-    const isOtherPathActive = () => {
-        if (props.OtherActivePages == undefined) return false;
-        for (const path of props.OtherActivePages!) {
-            if (location.pathname.includes(path)) return true;
-        }
-        return false;
-    }
```

Comment:
removing the logic for whether a Path is active seems like an issue.
Did we move that somewhere else?

Reply:
That logic was only used for the linkStyle, however we dont use linkStyle anymore and we let bootstrap handle the active page styling by adding the active class to the corresponding <a> tag.

Reply:
that's not quite right. We use bootstrap to apply the style (I think we use the `Active` class) However we need to retain the logic on when the page is active. If you hand that off to Bootstrap it won't work for a lot of pages because the logic in Bootstrap is very different.

Reply:
So you need to apply the `active` class whenever this logic get's triggered. For instance in SystemCenter the Meter Link is tied to the ByMeter page but it also shows active if someone is on the Meter page.